### ISSUE TITLE
Seeking Help with Splitting the Validation Set for Mediaeval2016

### ISSUE THREAD

**Issue by Qiaojiao-225** [MKLab-ITI/image-verification-corpus#6]
Hello, I used the Mediaeval2015 training set as my training set, the Mediaeval2015 test set as my validation set, and the Mediaeval2016 test set as my test set to prevent the occurrence of the same event samples in different splits. However, the results on the test set are noticeably worse compared to the training and validation sets. I suspect that this is due to a significant difference in data distribution between Mediaeval2015 and Mediaeval2016. Could you please suggest any good methods for splitting a validation set when using the Mediaeval2016 dataset?
